Neos.Eel/Classes/Helper/DateHelper.php

```diff
@@ -249,6 +249,39 @@ public function second(\DateTimeInterface $dateTime)
         return (integer)$dateTime->format('s');
     }
 
+    /**
+     * Format a given datetime in a machine-readable way to use it in a <time> tag
+     *
+     * @param \DateTimeInterface $dateTime
+     * @return string
+     */
+    public function htmlDateTime(\DateTimeInterface $dateTime)
+    {
+        return $dateTime->format(\DateTime::W3C);
```

Comment:
Is there a particular reason why you chose the `\DateTime:W3C` format?
I know that we use this as default in the `DateTimeConverter` but this was a mistake IMO because it lacks some crucial features.